static/app/views/alerts/incidentRules/metricField.tsx

```diff
@@ -76,7 +76,7 @@ const getFieldOptionConfig = ({
     })
   );
 
-  const fields = Object.fromEntries<ColumnType>(
```

Comment:
@JonasBa is there a performance penalty for the compilier for not using the generic here anymore?

Reply:
I dont think so. If any, these should be very small